### ISSUE TITLE
Feature to Scan ASN Number

### ISSUE THREAD

**Issue by Labs22** [robertdavidgraham/masscan#385]
Hey can you add a feature to scan the ASN number?

**Comment by Labs22** [comment]
This really help me a lot.
thank you very much
I'm in loving with IoT search engine.
I've been using shodan, censys, zoomeye and now Ivre.

**Comment by Rokin05** [comment]
```
ASNUM="XXXXXX"
whois -h whois.ripe.net -T route $ASNUM -i origin | egrep "route: " | awk '{print $NF}' >> MYASIP.TXT 
```

**Comment by p-l-** [comment]
@Rokin05 good point, thanks for this hint! (but you should try IVRE anyway ;-) )

You have (I think) an error: you need to add `AS` before `$ASNUM`: `whois -h whois.ripe.net -T route AS$ASNUM [...]` (at least, I had to do so). Also, you could use `awk` directly and remove `egreg`:

```Bash
ASNUM="XXXXXX"
whois -h whois.ripe.net -T route | awk '/^route:/ {print $NF}'
```
